src/app/productConfig/productConfigModal/productConfig.modal.component.js

```diff
@@ -235,6 +235,19 @@ class ProductConfigModalController {
       this.submittingGift = false;
     } );
   }
+
+  displayName() {
+    return this.productData.displayName === this.itemConfig['jcr-title'] ?
+      this.productData.displayName : this.itemConfig['jcr-title'];
+  }
+
+  displayId() {
+    let value = `#${this.productData.designationNumber}`;
+    if(this.itemConfig['jcr-title'] && this.productData.displayName !== this.itemConfig['jcr-title']) {
```

Comment:
In this scenario, if we used jcr-title instead of displayName, then we want to add displayName to the designationNumber line. We use jcr-title in place of displayName when they are different. I added jcr-title check since I didn't know if it would always be set, but I think it should be.